code/modules/antagonists/clockcult/clock_items/judicial_visor.dm

```diff
@@ -196,7 +196,7 @@
 				L.visible_message(span_warning("Strange energy flows into [L]'s [I.name]!"), \
 				span_userdanger("Your [I.name] shields you from [src]!"))
 			continue
-		L.Paralyze(15) //knocks down briefly when exploding
```

Comment:
This is already mostly fine, don't see any need to change it. The stun isn't very long but it can be chained into other stuff